### DISCUSSION TITLE
Getting to know hyper switch

### DISCUSSION THREAD

**Discussion by thunder-007** [juspay/hyperswitch#3638] [Q&A] [answered]
I'm learning some web development and trying to include payment gateway integration seen optios like razorpay stripe etc but I didn't liked the 2% charge they do.
"is hyperswitch simply a free selfhosted solution for payment gateway ?"
or am I missing something is there more to it..

**Comment by manojradhakrishnan** [comment, marked as answer]
Hyperswitch is not an alternative to Stripe or Razorpay. 

It is a unified API abstraction that helps to connect with multiple payment processors with minimal developer effort. Also helps to improve conversion rates and reduce payment costs by eliminating vendor lockin

You will need an account with payment processor to start using hyperswitch.

**Reply by thunder-007** [reply]
Thank you @manojradhakrishnan like I can integrate multiple payment gateways at once so let's say I have both American and indian users I can have both stripe and razor pay together and which ever gateway user uses I'll get charged their commission right?

**Reply by manojradhakrishnan** [reply]
Yes. That's correct.

**Comment by rudreshkoranne** [comment]
Hey @thunder-007,
I see you have a well understanding of the hyperswtich platform.
Well I'm trying to implement a custom wallet connector myself, But facing a lot of errors related to Application Configuration & req/resp implementation.

It would be a great help to me if you can guide me throught the things or provide resource that I can learn from.
Thanks!!!
